kraken-wrapper/poetry.lock

```diff
@@ -0,0 +1,900 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
How are we making sure this file stays up to date when our everyday workflow does not include Poetry?

Reply:
CI runs `poetry lock --check`